src/components/ButtonPanel.js

```diff
@@ -8,59 +9,45 @@ const styleGroup1 = {
   height: 100,
 };
 
-const Group1 = () => (
-  <div className="styleGroup1" style={styleGroup1}>
-    <Button buttonName="AC" color="#dfe8f7" />
-    <Button buttonName="+/-" color="#dfe8f7" />
-    <Button buttonName="A%" color="#dfe8f7" />
-    <Button buttonName="÷" rightB="yes" />
-  </div>
-);
-
-const Group2 = () => (
-  <div className="styleGroup2" style={styleGroup1}>
-    <Button buttonName="7" color="#dfe8f7" />
-    <Button buttonName="8" color="#dfe8f7" />
-    <Button buttonName="9" color="#dfe8f7" />
-    <Button buttonName="X" rightB="yes" />
-  </div>
-);
-
-const Group3 = () => (
-  <div style={styleGroup1}>
-    <Button buttonName="4" color="#dfe8f7" />
-    <Button buttonName="5" color="#dfe8f7" />
-    <Button buttonName="6" color="#dfe8f7" />
-    <Button buttonName="-" rightB="yes" />
-  </div>
-);
-
-
-const Group4 = () => (
-  <div className="styleGroup4" style={styleGroup1}>
-    <Button buttonName="1" color="#dfe8f7" />
-    <Button buttonName="2" color="#dfe8f7" />
-    <Button buttonName="3" color="#dfe8f7" />
-    <Button buttonName="+" rightB="yes" />
-  </div>
-);
-
-const Group5 = () => (
-  <div className="styleGroup5" style={styleGroup1}>
-    <Button buttonName="0" ZeroB="yes" color="#dfe8f7" wide />
-    <Button buttonName="." color="#dfe8f7" />
-    <Button buttonName="=" rightB="yes" />
-  </div>
-);
+const ButtonPanel = props => {
+  const { onClick } = props;
+  return (
+    <div className="style">
+      <div className="styleGroup1" style={styleGroup1}>
+        <Button buttonName="AC" onClick={onClick} color="#dfe8f7" />
```

Comment:
this is `<ButtonPanel onClick={this.handleClick} />` what you have in `app.js`

so you should pass the props to the bind function created in `app.js` 
example:
`<Button buttonName="AC" handleClick = { onClick } color="#dfe8f7" />`

Reply:
thanks